x-pack/plugins/cases/server/client/cases/get.ts

```diff
@@ -36,7 +36,6 @@ export const get = async ({
   try {
     let theCase: SavedObject<ESCaseAttributes>;
     let subCaseIds: string[] = [];
-
```

Comment:
I like the idea of having spaces before and after a new block of code starts. If you use Vim it make it very difficult to navigate between code blocks due to the missing of spaces.